backend/molgenis-emx2-sql/src/main/java/org/molgenis/emx2/sql/SqlDatabase.java

```diff
@@ -226,38 +230,6 @@ public void init() { // setup default stuff
     }
   }
 
-  private void initOidc() {
-    // oidc settings takes priority over env variables
-    String isOidcEnabledSetting = getSetting(Constants.IS_OIDC_ENABLED);
-    if (isOidcEnabledSetting != null) {
-      this.isOidcEnabled = Boolean.parseBoolean(isOidcEnabledSetting);
-    } else {
-      Object envSetting =
-          EnvironmentProperty.getParameter(Constants.MOLGENIS_OIDC_CLIENT_ID, null, STRING);
-      if (envSetting != null) {
-        this.setSetting(Constants.IS_OIDC_ENABLED, "true");
-        this.isOidcEnabled = true;
-      }
-    }
-    if (!this.isOidcEnabled) return;
-
-    // check if OIDC settings are complete otherwise log error and set to false
-    if (!isValidOidcSettings()) {
```

Comment:
So we don't check if OIDC settings are complete, when enabled? This is fine by me, but I made it because it was requested